### DISCUSSION TITLE
Add console eval helpers (`/$[_0-4]/`)

### DISCUSSION THREAD

**Discussion by jaril** [RecordReplay/devtools#5780] [Ideas]
Both [Chrome](https://developer.chrome.com/docs/devtools/console/utilities/) and [Firefox] DevTools(https://developer.mozilla.org/en-US/docs/Tools/Web_Console/Helpers) have user-facing console helpers/utilities. Making sure we support those in Replay seems reasonable, especially for frequently used ones.

We can start with:
- [ ] `$0`: Should evaluate to the currently-inspected element in the Elements Tab.
- [ ] `$_`: Should evaluate to the result of the last evaluated expression in the console.

**Comment by abstractalgo** [comment]
Per https://developer.chrome.com/docs/devtools/console/utilities/#recent-many, it can go from `$0` to `$4`.
